### ISSUE TITLE
Issue to render properly configured HAR

### ISSUE THREAD

**Issue by vic0707** [micmro/PerfCascade#162]
When trying to render the attached har file using [PerfCascade](https://micmro.github.io/PerfCascade/), I get the following message in chrome's console and nothing is displayed: 

```
TypeError: Cannot read property 'length' of undefined
    at Object.f [as transformDoc] (perf-cascade.min.js:3)
    at Object.e [as fromHar] (perf-cascade.min.js:2)
    at renderPerfCascadeChart (demo-page.js:38)
    at demo-page.js:72
    at d (perf-cascade-file-reader.min.js:3)
    at FileReader.<anonymous> (perf-cascade-file-reader.min.js:3)
```

I believe the issue is related to the fact that the pages field is missing from the HAR but this should not be blocking as pages is an optional field, as can be seen in the [specs](http://www.softwareishard.com/blog/har-12-spec/#log)

[cnn.har.zip](https://github.com/micmro/PerfCascade/files/797105/cnn.har.zip)

Thanks

**Comment by micmro** [comment]
It looks like it misses the [`pages`](http://www.softwareishard.com/blog/har-12-spec/#log) attribute (which is valid HAR).
I had only looked at wpt and chrome, which always add it. Will add some logic to handle this case.

**Comment by micmro** [comment]
fixed with `v0.6.3`
